src/components/common/YourCompany/YourCompany.vue

```diff
@@ -458,19 +454,14 @@ export default class YourCompany extends Mixins(
     return this.toDisplayPhone(this.nrApplicant.phoneNumber)
   }
 
-  /** The recognition date or business start date-time string. */
-  get recognitionDateTime (): string {
+  /** The recognition date or business start date string. */
+  protected get recognitionDateTime (): string {
```

Comment:
_Probably_ getters shouldn't be called externally, but there's no harm in them being public (default when no modifier is present), since they only expose data, not change it.

Reply:
Done